server/coordinator/procedure/ddl/createtable/create_table.go

```diff
@@ -190,17 +201,9 @@ func (p *Procedure) Start(ctx context.Context) error {
 		p:   p,
 	}
 
-	if err := p.fsm.Event(eventPrepare, req); err != nil {
-		err1 := p.fsm.Event(eventFailed, req)
+	if err := p.fsm.Event(eventCreateTable, req); err != nil {
```

Comment:
Maybe we can define a `FsmError` to judge whether it is an error in the internal execution of the state machine, so as to decide whether it can be retried.
```
if err := p.fsm.Event(eventCreateTable, req); err != nil {
switch err.(type){
 case: FsmError:
    if retrySize < MaxRetrySize{
       p.fsm.Event(eventCreateTable, req)
     }else{
      p.fsm.Event(eventRollback,req)
}
....
}
```